### DISCUSSION TITLE
Presenting more detailed schema information about mesurements in a single command

### DISCUSSION THREAD

**Discussion by StepY1aoZz** [openGemini/openGemini#706] [Ideas]
As issue #640 said, to obtain a comprehensive view of the measurements schema, users must combine many commands, such as `SHOW TAG KEYS`, `SHOW FIELD KEYS` and `SHOW SCHEMA`, to achieve that.  Also, the retention policy is lacking in the return value of `SHOW MEASUREMENTS` command.

Thus, I believe that it would be better to have a new command, which can help users to get their wanted infomation in a single command. A possible implement could be like this:

**Command name:** `SHOW MEASUREMENTS DETAIL`
**Command syntax:**  `SHOW MEASUREMENTS DETAIL [ON database] [WITH measurement <operator> ['<measurement_name>' | <regular_expression>]]`
**Command return value:**
including the measurement name, retention policy name, tag keys (result of `SHOW TAG KEYS`),  field keys (result of `SHOW FIELD KEYS`), and schema (result of `SHOW SCHEMA`) of measurements.
![图片](https://github.com/user-attachments/assets/31d5af03-8436-4ea8-8d6e-2a6c308ea830)

**Comment by xiangyu5632** [comment]
ref #710
